decode/api/fast.py

```diff
@@ -11,6 +11,7 @@
 from decode.ml_logic.face_detection.main import get_face_detection_model
 
 app = FastAPI()
+
 model_face_detection = get_yolo_model()
```

Comment:
don't you have to implement the name app.state.model_yolo  or app.state.model_emotion?